.changeset/brave-jobs-invite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-common': patch
+---
+
+Added a new `LegacyDatabaseManager` interface that can be used to avoid direct dependencies on the `DatabaseManager`.
```

Comment:
```suggestion
Added a new `LegacyDatabaseService` interface that can be used to avoid direct dependencies on the `DatabaseManager`.
```

But really, SHOULD it be named `LegacyDatabaseManager` instead? Since DatabaseManager.forPlugin->DatabaseService

Reply:
Yep was a bit uncertain about this too. Idea behind service is that this is a service after all, maybe `LegacyRootDatabaseService`? Also figured it doesn't matter too much since it should go away.